### PR TITLE
B3 style template cleanup

### PR DESCRIPTION
**Pull request by biyeun**
- removes b2/b3 splits of templates in `style`
- removes b2/b3 split in mocha templates

@orangejenny / @czue cc @calellowitz

### CODE REVIEW

**Comment by biyeun** [comment]
screen shot 2016-06-01 at 11 31 28 am

Yeah, that's right. 🎉

File: settings.py [hunk 1]
```diff
@@ -470,7 +470,7 @@
 ####### Shared/Global/UI Settings #######
 
 # restyle some templates
-BASE_TEMPLATE = "style/bootstrap3/base.html"  # should eventually be bootstrap3
+BASE_TEMPLATE = "style/base.html"  # should eventually be bootstrap3
```
Comment (orangejenny) [inline settings.py]: might as well drop the comment for clarity

File: corehq/apps/style/templates/style/partials/form_list_form.html [hunk 2]
```diff
@@ -3,7 +3,7 @@
 {% load i18n %}
 
 <form id="table-form"
-      action="."
+      action="../bootstrap3/partials"
```
Comment (orangejenny) [inline corehq/apps/style/templates/style/partials/form_list_form.html]: Curious, what's this? Doesn't this imply that there's a `bootstrap3/partials` directory somewhere in `style` (don't see one)?

**Comment by orangejenny** [comment]
Looks reasonable, waiting on tests. Left one question.

**Comment by orangejenny** [comment]
reminder, merge https://github.com/dimagi/ctable/pull/91 along with this

File: corehq/apps/style/templates/style/partials/form_list_form.html (hunk 2 above)
Comment (biyeun) [inline corehq/apps/style/templates/style/partials/form_list_form.html]: i have a feeling that might have accidentally gotten in there crap due to a command+c mishap. thanks for the catch

**Comment by orangejenny** [comment]
has conflicts

**Comment by biyeun** [comment]
@orangejenny rebased master into branch and build has restarted ((sigh))

**Comment by biyeun** [comment]
test failure due to submodule not being committed. :sob:

**Comment by orangejenny** [comment]
awww

**Comment by orangejenny** [comment]
more conflicts :-/

**Comment by biyeun** [comment]
!*(@$^!*&$Y!(&!

**Comment by biyeun** [comment]
*waits hours for build to complete again. only to find more merge conflicts*

**Comment by biyeun** [comment]
:sob:

**Comment by biyeun** [comment]
someone added a new mocha test in the meantime and build fail :sob: *sigh*

**Comment by biyeun** [comment]
YAAAAAAAY @orangejenny ✅

**Comment by biyeun** [comment]
fyi @czue ✅

**Comment by proteusvacuum** [comment]
:tada:

**Comment by orangejenny** [comment]
Yes! congrats @biyeun

**Comment by czue** [comment]
woohoo!

**Pull request merged**